templates/user/settings/security/accountlinks.tmpl

```diff
@@ -1,5 +1,25 @@
 <h4 class="ui top attached header">
 	{{.i18n.Tr "settings.manage_account_links"}}
+	{{if .OrderedOAuth2Names}}
+		<div class="ui right">
+			<div class="ui floating1 jump dropdown">
+				<div class="ui blue tiny button">Link Account</div>
```

Comment:
```suggestion
			<div class="ui dropdown">
				<div class="ui primary tiny button">Link Account</div>
```

The `floating1 jump` seems to be copied from somewhere else, it seems incorrect.

`blue` is not a cross-theme CSS name, use `primary` instead (the main branch already has the refactoring)